object-cache.php

```diff
@@ -156,13 +159,28 @@ function add( $id, $data, $group = 'default', $expire = 0 ) {
 			$expire = $this->default_expiration;
 		}
 
+		if ( is_string( $data ) ) {
+			$size = strlen( $data );
+		} else {
+			$serialized = serialize( $data );
+			$size = strlen( $serialized );
+		}
```

Comment:
Thoughts on moving this to a helper function since we repeat it several times.

Reply:
Done.